packages/ui-components/src/utils/clipboard.helper.ts

```diff
@@ -0,0 +1,52 @@
+import { TooltipPosition } from '../components/tooltip/tooltip.types';
+
+const bundle = {
+	fallbackCopyTextToClipboard,
+	copyTextToClipboard
+};
+
+export const COPY_TOOLTIP: {
```

Comment:
can this be moved to the place where it's used inside a config file instead of being here or is it strictly necessary to be here?